### PR TITLE
[WIP] adding v4 documentation and v3 deprecation

### PR DESCRIPTION
**Pull request by krazybean**

### CODE REVIEW

File: source/api-email-validation.rst [hunk 1]
```diff
@@ -213,3 +219,224 @@ Attaching to a form:
...
+did_you_mean             string       (Optional) Null if nothing, however if a potential typo is made, the closest suggestion is provided
+is_disposable_address    boolean      If the domain is in a list of disposable email addresses, this will be appropriately categorized
+is_role_address          boolean      Checks the mailbox portion of the email if it matches a specific role type ('admin', 'sales', 'webmaster')
+reason                   array        List of potential reasons why a specific validation may be unsuccessful.
+result                   string       Either "deliverable", "undeliverable" or "unknown" status given the evaluation.
+risk                     string       "High", "Medium", "Low" or "null" Depending on the evaluation of all aspects of the given email.
+=====================    =========    ============================================================================================================
+
+
+Version 4 List Validation
```
Comment (aerwin3) [inline source/api-email-validation.rst]: Due to this being a /v3 endpoint this can be confusing.

File: source/api-email-validation.rst [hunk 2]
```diff
@@ -213,3 +219,224 @@ Attaching to a form:
...
+
+.. code-block:: javascript
+
+    {
+      "created_at": "Tue, 26 Feb 2019 21:30:03 GMT",
+      "download_url": {
+        "csv": "<download_link>",
+        "json": "<download_link>"
+      },
+      "id": "list_validations@sandbox.mailgun.org",
```
Comment (aerwin3) [inline source/api-email-validation.rst]: maybe change this id so list and bulk validations don't get confused.

File: source/api-email-validation.rst [hunk 3]
```diff
@@ -213,3 +219,224 @@ Attaching to a form:
...
+
+Example of successful mailbox verification result.
+
+.. code-block:: javascript
+
+    {
+        "address": "existingemail@realdomain.com",
+        "is_disposable_address": false,
+        "is_role_address": false,
+        "reason": [],
```
Comment (thrawn01) [inline source/api-email-validation.rst]: `reason` field changed types between a failed result and an successful result.

File: source/api-email-validation.rst [hunk 4]
```diff
@@ -213,3 +219,224 @@ Attaching to a form:
...
+
+.. code-block:: javascript
+
+    {
+      "id": "57713334-aa10-4699-a1fe-1c624ca6a671",
+      "message": "The validation job was submitted."
+    }
+
+
+V4 List Validation Response Example
```
Comment (thrawn01) [inline source/api-email-validation.rst]: This says v4, but the urls are v3

Comment (krazybean) [reply source/api-email-validation.rst]: Thanks, this is the same that @aerwin3, will have fixed here shortly, thank you for your review.

File: source/api-email-validation.rst (hunk 3 above)
Comment (krazybean) [reply source/api-email-validation.rst]: Spoke with @aerwin3 about this, we are going to make a change so that its consistently a list/array even if empty

**Comment by krazybean** [comment]
Documentation has been updated to the latest feedback, any additional feedback is welcomed and appreciated!

**Pull request merged**